tests/config/pytest_modules.yml

```diff
@@ -2467,3 +2474,6 @@ yara/index:
 yara/mapper:
   - modules/yara/mapper/**
   - tests/modules/yara/mapper/**
+
+__anchors__:
```

Comment:
Just for my understanding, what's the purpose of the explicit anchor?

Reply:
To be able to maintain the anchors in the file when we update it with tools, we need to add the aliases. I think that grouping all of them under the same name can be easier to maintain. 
The only modification that I found while adding the functionality in tools is that we would  need to rename `__anchors__` so it doesn't get mixed with other modules, as we sort them alphabetically, I would suggest `~anchors:` which will make it be last in the Python sorting.